benchexec/tools/korn.py

```diff
@@ -0,0 +1,52 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+import re
+import benchexec.util as util
+import benchexec.result as result
+import benchexec.tools.template
+
+
+class Tool(benchexec.tools.template.BaseTool):
```

Comment:
Would you mind switching to our new `BaseTool2` API? ([migration guide](https://github.com/sosy-lab/benchexec/blob/master/doc/tool-integration.md#migrating-tool-info-modules-to-new-api)) Should be only little effort and could for example make `determine_result` even easier.

Reply:
thanks, I've migrated it to the new API